server/services/WalletService.js

```diff
@@ -46,6 +48,12 @@ class WalletService {
 
       await this._session.commitTransaction();
 
+      await this._event.logEvent({
+        loggedInWalletId: addedWallet.id,
```

Comment:
addedWallet.id is not the loggedInWalletId

Reply:
@Kpoke, I was thinking that the newly created wallet will have a log that says when it was created. Should we add this for the loggedInWalletId as well ?

Reply:
Added this

![Screenshot 2023-09-21 at 12 12 17 AM](https://github.com/Greenstand/treetracker-wallet-api/assets/104117557/ef6bc427-ee49-4d88-a7cb-5daba3dcaf04)

Reply:
LoggedInWalletId should be the same for both instances. Which is the id of the wallet that is logged in/parent wallet.

Reply:
Also, the type of parentWallet/childWallet should be uniform. Let's use the wallet name for both

Reply:
So I think the issue here is the naming of loggedInWalletId, I can see in the EventModel file it is actually wallet_id. So let's expose it as wallet_id from the model file and not loggedInWalletId, for cases like this where we might want to add a log for multiple wallets.

Reply:
changed all the loggedInWalletId to wallet_id